atom/browser/resources/win/atom.rc

```diff
@@ -56,8 +56,8 @@ END
 //
 
 VS_VERSION_INFO VERSIONINFO
- FILEVERSION 3,1,8,0
- PRODUCTVERSION 3,1,8,0

```

Comment:
@harryi3t Does this signify the point version in the pre-release block?

Reply:
@kamalaknn  What's a `point version`?
Here, the first 3 numbers are the major, minor & patch version respectively
The last number is gotten by splitting the pre-release section by dot and taking the first number
eg: `x.y.z-someString.a.b.c` => here `a` would be taken as 4th number

Reply:
That's what I meant. Also why only the first number?

Reply:
@kamalaknn Windows application only supports 4 digit format. Electron chose to keep the first one as the 4th digit.